src/components/app/reducers.js

```diff
@@ -0,0 +1,32 @@
+import { SET_COOKIE, REMOVE_COOKIE } from './actionTypes';
+
+const initialState = {
+    loggedIn: false,
+    user: "",
+};
+
+export default ( state = { ...initialState }, action) => {
+    switch (action.type) {
+    case SET_COOKIE:    {
+        const time = 12;
+        const expiration = new Date(new Date().getTime()+ time * 3600 * 1000);
+        const token = action.data.token || 'testtoken';
+        browser.cookies.set(token, expiration);
```

Comment:
why are you still using `browser` here. Did we decided that you would be using `cookies-js`?